app/src/main/java/com/codepath/daggerexample/MainActivity.java

```diff
@@ -25,8 +31,8 @@
     @Inject
     Retrofit mRetrofit;
 
-    private GitHubApi mGitHubApi;
```

Comment:
the docs and the code are not in alignment... https://github.com/codepath/android_guides/wiki/Dependency-Injection-with-Dagger-2, should this be made an issue?